business-resources/green-leaf-program.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: 'Green Leaf Program'
+---
+
+[**Return to Homepage**](/)
+
+# Green Leaf Business Recognition Program
+
+The Borough of Rutherford, as a Sustainable Jersey Silver Certified Community, has made a commitment to support the economic, cultural and environmental sustainability of the community. Rutherford will be announcing a new local sustainable business recognition program. The Green Leaf Recognition Program will assist businesses to make a similar commitment while promoting your business, saving money, minimizing risk, gaining competitive advantage and being socially responsible.
+
+As a preview to the Green Leaf Recognition program, some of the criteria used to assess sustainability will be based on the Sustainable Jersey green business recognition action, including:
+
+1. Water efficiency in kitchens and restrooms
+2. Waste reduction and recycling of food and materials
```

Comment:
Nit - these can all be

1.

That way changing order or adding a new one in the future won't require renumbering in the markdown. The md=>html converter does the right thing and will number them correctly.